### ISSUE TITLE
Create keystone app installs incompatible node_modules

### ISSUE THREAD

**Issue by garth** [keystonejs/keystone#2653]
# Bug report

## Describe the bug

create keystone app installs and old version of keystone but latest versions of keystone plugins.

## To Reproduce

Steps to reproduce the behaviour. Please provide code snippets or a repository:

### install

```
yarn create keystone-app jotsec
yarn create v1.21.1
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "create-keystone-app@1.7.0" with binaries:
      - create-keystone-app
[##############################################################] 62/62

 ╦╔═ ╔═╗ ╦ ╦ ╔═╗ ╔╦╗ ╔═╗ ╔╗╔ ╔═╗  ╦ ╔═╗ 
 ╠╩╗ ║╣  ╚╦╝ ╚═╗  ║  ║ ║ ║║║ ║╣   ║ ╚═╗ 
 ╩ ╩ ╚═╝  ╩  ╚═╝  ╩  ╚═╝ ╝╚╝ ╚═╝ ╚╝ ╚═╝ 


Answer a few questions and we'll generate a starter project for you.

✔ What is your project name? … jotsec
✔ Select a starter project › Starter (Users + Authentication)
✔ Select an adapter › Knex
✔ Copying project files
Installing dependencies with yarn. This could take a few minutes.
✔ Installed dependencies

  🎉  KeystoneJS created a starter project in: jotsec

  To launch your app, run:

  - cd jotsec
  - yarn dev

  Next steps:

  - View your app
  - Edit jotsec/index.js to customize your app.
  - Open the Admin UI
  - Read the docs
  - Star KeystoneJS on GitHub

Done in 138.61s.
```

### Create a database

create a database and add DATABASE_URL env

### Start keystone

```
yarn dev
ℹ Command: keystone dev
✔ Validated project entry file ./index.js
✔ Keystone server listening on port 3000
✔ Initialised Keystone instance
✖ Connecting to database
{ Error: Cannot read property 'find' of undefined
    at Promise.all.then.results (/home/garth/tmp/jotsec/node_modules/@keystonejs/utils/dist/utils.cjs.dev.js:48:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errors:
   { KnexAdapter:
      TypeError: Cannot read property 'find' of undefined
          at fieldAdapters.forEach.fieldAdapter (/home/garth/tmp/jotsec/node_modules/@keystonejs/adapter-knex/lib/adapter-knex.js:258:31)
          at Array.forEach (<anonymous>)
          at KnexListAdapter._postConnect (/home/garth/tmp/jotsec/node_modules/@keystonejs/adapter-knex/lib/adapter-knex.js:257:24)
          at Object.values.forEach.listAdapter (/home/garth/tmp/jotsec/node_modules/@keystonejs/adapter-knex/lib/adapter-knex.js:73:19)
          at Array.forEach (<anonymous>)
          at KnexAdapter.postConnect (/home/garth/tmp/jotsec/node_modules/@keystonejs/adapter-knex/lib/adapter-knex.js:72:38)
          at KnexAdapter.connect (/home/garth/tmp/jotsec/node_modules/@keystonejs/adapter-knex/node_modules/@keystonejs/keystone/lib/adapters/index.js:29:38)
          at process._tickCallback (internal/process/next_tick.js:68:7) } }
```

### list outdated packages

```
yarn outdated v1.21.1
info Color legend : 
 "<red>"    : Major Update backward-incompatible updates 
 "<yellow>" : Minor Update backward-compatible features 
 "<green>"  : Patch Update backward-compatible bug fixes
Package              Current Wanted Latest Package Type URL                                                                 
@keystonejs/keystone 6.0.2   6.0.2  7.0.0  dependencies https://github.com/keystonejs/keystone/tree/master/packages/keystone
Done in 3.46s.
```

## Expected behaviour

All packages should be the same major version (currently v7). After keystone is updated to v7 it starts without error.

## System information

tested on WSL2 (Ubuntu) & Arch

**Comment by thekevinbrown** [comment]
This appears to be a duplicate of #2588.

**Comment by timleslie** [comment]
This has been fixed in version `3.0.0` of `create-keystone-app`. Please try running the script at this new version and by all means re-open this ticket if you find the same/similar issues with the latest version.
